tasks/package.js

```diff
@@ -96,9 +168,12 @@ module.exports = function(grunt) {
       }
     });
 
-    grunt.config.set('clean.packages', [destPath]);
-
-    tasks.push('clean:packages');
+    if (useRsync) {
+      tasks.push('mkdir:package');
+      tasks.push('shell:rsync');
+    } else {
+      tasks.push('copy:source');
+    }
     tasks.push('copy:package');
```

Comment:
copy:package looks like it's been stripped down to some final copy operations of random project files. Maybe this should be renamed for clarity? `copy:package-extras`?